File: cpp/src/pixel.cpp
#include"pixel.h"
Pixel::Pixel()
{
	this->R = this->G = this->B = 0;
}
Pixel::Pixel(Point& _pos, int& _data)
{
	this->pos 	= _pos;
	this->R = this->G = this->B = 0;
}

File: python/QuadTree/Pixel.py
from QuadTree.Point import Point

class Pixel(object):
	def __init__(self, color = [0, 0, 0],
			topLeft = Point(0, 0),
			bottomRight = Point(0, 0)):
		self.R = color[0]
		self.G = color[1]
		self.B = color[2]
		self.topLeft = topLeft
		self.bottomRight = bottomRight
